app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('miq.charts').directive('heatMap', ['$timeout',
+  function($timeout) {
```

Comment:
> it will be good not to copy paste code if possible to nicely import it through the new release. @dclarizio @skateman, would that be possible in this case?

Ping @dclarizio\@skateman